### DISCUSSION TITLE
My app reports "Client with token "xxx..." not recognised

### DISCUSSION THREAD

**Discussion by rdevcic** [tidal-music/discussions#52] [Q&A]
After more then 3 years of working App with Tidal integration - today I got error message "Client_ID is not recognised by Tidal" any more. This results not able to log in to Tidal services by valid subscription.
I made new, client_id/client_secret on developer portal. Yet I got another error: "Client with "Client_ID" is not a limited Input Device Client.  Interested what shall be done to get Tidal working again in my app?

**Comment by stefanmoro** [comment]
Hey @rdevcic ,

It sounds to me like you are the maintainer of a previously existing partner integration? If so, I suggest you reach out to your contact partner contact at TIDAL.

Thanks
